source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -117,7 +128,10 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
         button.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
         button.topAnchor.constraint(equalTo: topAnchor).isActive = true
         button.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
-        buttonLabelField.leadingAnchor.constraint(equalTo: button.trailingAnchor, constant: buttonConfig?.elementSpacing ?? 8).isActive = true
+        button.heightAnchor.constraint(greaterThanOrEqualToConstant: CGFloat(16)).isActive = true
+        button.widthAnchor.constraint(equalToConstant: CGFloat(16)).isActive = true
```

Comment:
Do you need these 2 even when compressionresistance is set to .required?

Reply:
Yes, without that sometime button is slightly clipped. we have set setContentCompressionResistancePriority for occupancy needed space from right side textview.
<img width="105" alt="Screenshot 2022-05-24 at 1 14 54 PM" src="https://user-images.githubusercontent.com/105660080/169976972-ddeb22f9-ce22-4537-a0be-189fe0d97f6a.png">